share/spice/js_keycodes/content.handlebars

```diff
@@ -0,0 +1,25 @@
+{{#if result}}
+<p class="keycode-result">The keycode for <code>{{result.key}}</code> is: {{result.code}}</p>
+{{else}}
+<p class="keycode-result">Press a key in the textbox to check its keycode:</p>
+<input type="text" class="keycode-input"/>
+{{/if}}
+<a class="btn btn--primary show-keycodes" href="#">Show all codes</a>
+
+{{! hidden when the page first renders }}
+<table class="all-keycodes">
```

Comment:
Might be best to add the class "is-hidden" here so that it doesn't need to wait for the Spice's CSS to load in order to hide it, or you could use inline css but the former is better :smiley: